lib/browser/rpc-server.js

```diff
@@ -453,12 +457,28 @@ const crashReporterInit = function (options) {
   }
 }
 
-ipcMain.on('ELECTRON_CRASH_REPORTER_INIT', function (event, options) {
+const setReturnValue = function (event, getValue) {
   try {
-    event.returnValue = [null, crashReporterInit(options)]
+    event.returnValue = [null, getValue()]
   } catch (error) {
     event.returnValue = [errorUtils.serialize(error)]
   }
+}
+
+ipcMain.on('ELECTRON_CRASH_REPORTER_INIT', function (event, options) {
```

Comment:
For other reviewers these event handlers were moved here from elsewhere, they are not added in this PR